app/templates/sign-in.html

```diff
@@ -56,11 +59,12 @@ <h1 class="saturn">Sign in</h1>
                     </span>
                     {{ form.password(class_='input input--text input-type__input js-password-obfuscation-field') }}
 
-                    <a href="/forgot-password">Forgot password</a>
+                    <br />
+                    <a href="/forgot-password">Forgot password?</a>
                 </div>
                 <br />
 
-                <button class="btn btn--primary venus" type="submit">Sign in</button>
+                <button class="btn btn--primary venus" type="submit" id="SIGN_IN_BUTTON">Sign in</button>
             {% if errorType == "failed" %}
```

Comment:
separate issue from this pull request but should sign-in trigger this error stuff when it's submitted empty as well?